<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingMenu</class>
 <widget class="QWidget" name="DrawingMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="m_DrawingTools">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Outils de dessin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="m_ToolLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="m_PenButton">
          <property name="text">
           <string>Crayon</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="m_EraserButton">
          <property name="text">
           <string>Gomme</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="m_PingButton">
          <property name="text">
           <string>Ping</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="m_OptionWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="m_PenPage">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Taile</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_PenSpinBox">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Couleur</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ColorSelectionWidget" name="m_ColorSelectionWidget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="m_EraserPage">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Taille</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_EraserSpinBox">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="m_PingPage"/>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="finalLayout">
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_EraseButton">
          <property name="text">
           <string>Effacer dessins</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_SendButton">
          <property name="text">
           <string>Envoyer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">Canvas/SideMenuWidgets/ColorSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
